File: python/tr16.py
valor_compra = float(input("Ingrese el valor de la compra: "))

if valor_compra >= 100000:
    descuento = valor_compra * 0.10
    total_pagar = valor_compra - descuento
    print(f"Se aplica un descuento del 10%. Total a pagar: {total_pagar}")
else:
    print(f"No hay descuento. Total a pagar: {valor_compra}")

File: java script/tr16.js
let compra = prompt("Ingrese el valor de la compra");
if (compra >= 100000) {
    let descuento = compra * 0.10;
    let total = compra - descuento;
    alert("Tiene descuento.\nTotal a pagar: " + total);
} else {
    alert("No tiene descuento.\nTotal a pagar: " + compra);
}